fiftyone/core/view.py

```diff
@@ -123,6 +129,7 @@ def __copy__(self):
             _stages=deepcopy(self.__stages),
             _media_type=self.__media_type,
             _group_slice=self.__group_slice,
+            _name=self.__name,
```

Comment:
I included name in `__copy__` implementation with the idea that copying a saved view is still the same saved view